src/herfindahlIndex.js

```diff
@@ -0,0 +1,24 @@
+var enforceNumber = require('./utils/enforce/number');
+
+/**
+ * Calculates Herfindahl-Hirschman Index, a measure of market concentration and competition, used by U.S. governments for antitrust enforcement.
+ * @param {array} marketShare (expressed as percentage value for each firm)
+ * @returns {number} Herfindahl-Hirschman Index (HHI), a number between 0 and 10,000
+ */
+
+function herfindahlIndex(marketShare) {
+  var result = 0;
+
+  enforceNumber(marketShare);
```

Comment:
yeah I wondered. so is the same thing breaking here? https://github.com/CapitalJS/capitaljs/blob/master/src/returnOnInvestment.js#L12